.github/workflows/test.yml

```diff
@@ -95,8 +92,9 @@ jobs:
             experimental: false
             parallel: '--parallel'
           - python: '3.10'
-            django: 'git+https://github.com/django/django.git@stable/5.0.x#egg=Django'
-            experimental: true
+            django: 'Django>=5.0,<5.1'
+            experimental: false
+            parallel: '--parallel'
```

Comment:
We need to ensure there's at least one non-parallel job for each database in case of unreadable tracebacks, so this shouldn't be parallel. Also, per the comment in the yml file, we should keep testing against `stable/5.0.x` instead of completely replacing it.